packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -640,6 +666,15 @@ export class FluidDataStoreRuntime
 		raiseConnectedEvent(this.logger, this, connected, clientId);
 	}
 
+	private _readonly: boolean;
+	public setReadOnlyState(readonly: boolean) {
```

Comment:
Since the `FluidDataStoreRuntime` class is unfortunately still exposed as `legacy`, this method is visible/callable by legacy customers.  Maybe at least add a doc comment warning them away from calling it.